client/components/domains/trademark-claims-notice/index.jsx

```diff
@@ -0,0 +1,563 @@
+						</li>
+					) ) }
+				</ul>
+			</div>
+		);
+	};
+
+	renderContactInfo = contact => {
+		if ( ! contact ) {
+			return;
+		}
+
+		const { addr } = contact;
+		if ( addr.street && ! Array.isArray( addr.street ) ) {
+			addr.street = [ addr.street ];
+		}
+
+		return (
+			<ul className="trademark-claims-notice__claim-item-list">
+				{ contact.name && (
+					<li key="name">
+						<span className="trademark-claims-notice__claim-item-label">Name: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ contact.name }</span>
+					</li>
+				) }
+				{ contact.org && (
+					<li key="org">
+						<span className="trademark-claims-notice__claim-item-label">Organization: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ contact.org }</span>
+					</li>
+				) }
+				{ addr.street &&
+					addr.street.map( ( street, index ) => (
+						<li key={ 'street' + index }>
+							<span className="trademark-claims-notice__claim-item-label">Address: </span>
+							<span className="trademark-claims-notice__claim-item-data">{ street }</span>
+						</li>
+					) ) }
+				{ addr.city && (
+					<li key="city">
+						<span className="trademark-claims-notice__claim-item-label">City: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ addr.city }</span>
+					</li>
+				) }
+				{ addr.sp && (
+					<li key="sp">
+						<span className="trademark-claims-notice__claim-item-label">State: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ addr.sp }</span>
+					</li>
+				) }
+				{ addr.pc && (
+					<li key="pc">
+						<span className="trademark-claims-notice__claim-item-label">Postal Code: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ addr.pc }</span>
+					</li>
+				) }
+				{ addr.cc && (
+					<li key="cc">
+						<span className="trademark-claims-notice__claim-item-label">Country: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ addr.cc }</span>
+					</li>
+				) }
+				{ contact.voice && (
+					<li key="voice">
+						<span className="trademark-claims-notice__claim-item-label">Phone: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ contact.voice }</span>
+					</li>
+				) }
+				{ contact.fax && (
+					<li key="fax">
+						<span className="trademark-claims-notice__claim-item-label">Fax: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ contact.fax }</span>
+					</li>
+				) }
+				{ contact.email && (
+					<li key="email">
+						<span className="trademark-claims-notice__claim-item-label">Email: </span>
+						<span className="trademark-claims-notice__claim-item-data">{ contact.email }</span>
+					</li>
+				) }
+			</ul>
+		);
+	};
+
+	renderTrademarkClaimRegistrant = claim => {
+		const { holder } = claim;
+
+		if ( ! holder ) {
+			return;
+		}
+
+		const { addr } = holder;
+		if ( addr.street && ! Array.isArray( addr.street ) ) {
```

Comment:
Seems we do this here and in the next function, and similar stuff for other fields. Feels like maybe we could put this logic somewhere higher - either as soon as we get this data or even on the backend? Doesn't seem like something that the component should be aware of - `street` is an array and that's it.